.github/workflows/on_PR_linux_special_builds.yml

```diff
@@ -55,7 +55,8 @@ jobs:
         run: |
           cd build
           ctest --output-on-failure
-          gcovr --root .. --object-dir . --exclude-directories xmpsdk --exclude-directories unitTests --exclude-directories samples --exclude '.*xmpsdk.*' --exclude '.*unitTests.*' --exclude '.*samples.*' --exclude-unreachable-branches --exclude-throw-branches --xml -o coverage.xml
```

Comment:
@piponazo I think the fact that this wasn't identical to what we do for the `main` branch might be the cause for the weird coverage reports we see.   

I've just removed the excludes such that we collect any coverage information we can find. 
The reporting will be steered by the new `codecov.yml` to only report info about `src/*` and `include/*`

Reply:
ok, granted I had to fiddle one more commit into this PR, but now the coverage report above looks like what I'd expect.